### DISCUSSION TITLE
Dash Problem - The librosa plot doesn't appear

### DISCUSSION THREAD

**Discussion by ELITSZORD** [teamalgoritma/community#700] [Q&A] [answered]
Dear all members,

Saya ada problem sebagai berikut:
1. Menampilkan plot librosa lib: Ketika ke halaman audio analysis, kemudian file wav di upload. Jika kita hanya mengoperasikan kode untuk menampilkan plot waveform dan plot energy, maka di halaman  audio analysis plot wave form dan energy akan muncul. Apabila kode ditambahkan untuk memunculkan plot pitch dan mfcc dimana ada 4 plot, akan tetapi ke 4 plot tidak muncul atau muncul terkadang berantakan.

2. Ketika navbar overview result di tekan, kemudian kembali ke home page, isi dari halaman overview result ikut kedalam halaman home page dimana seharusnya halaman overview result tidak ikut.

3. The navbar size tidak fit ke ukuran screen.

Teman-teman bisa akses video rekaman dashboard saya agar terlihat bagaimana problemnya dan disertai file pyscript saya di link berikut : https://drive.google.com/drive/folders/1tYKC3bb4zx63XQ5-S9BY1wUdxvL4HQJE?usp=sharing

Mohon bantuannya teman2 karena sudah stuck lama akan hal ini. 
Thank you.

**Comment by IrfanChairurrachman** [comment]
Dear @ELITSZORD

Terima kasih atas pertanyaannya, akan kami coba telusuri ya Pak permasalahannya.

**Reply by ELITSZORD** [reply, marked as answer]
@IrfanChairurrachman Terimakasih mas atas responnya, kemarin ketika saya share di forum dash sudah di jawab tidak lebih dari 2 hari. Jadi solusinya bisa menggunakan state dimana dia bisa menyimpan suatu nilai. Dan saran kedua dari mba @dwiknrd tidak usah di pisah2 callbacknya jadi di jadikan satu saja (akan tetapi lebih aman pake state). Thanks mas Irfan.
